controller/server/controller.go

```diff
@@ -263,9 +263,9 @@ func (c *Controller) Run() {
 		pfxlog.Logger().Fatalf("failed to create Edge Client API factory: %v", err)
 	}
 
-	if err = c.policyEngine.Start(c.AppEnv.HostController.GetCloseNotifyChannel()); err != nil {
-		log.WithError(err).Fatalf("error starting policy engine")
-	}
+	//if err = c.policyEngine.Start(c.AppEnv.HostController.GetCloseNotifyChannel()); err != nil {
```

Comment:
Commented out?

Reply:
For testing purposes, need to keep the massive set of expired api tokens around. Will put back in before merging. Please don't approve until it's back in ;)

Reply:
Too late, approved =(

Reply:
Nooooooooo!!

Reply:
We need some marker file that will fail the build with reminders of things to do before merge